src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -36,3 +52,50 @@
         outline: auto;
     }
 }
+
+.bcs-ActivityItem.bcs-is-focused .bcs-ActivityThread {
+    padding-bottom: 0;
+    box-shadow: none;
+
+    &,
+    &:hover {
+        background-color: $white;
+    }
+
+    .bcs-ActivityThread-selectWrapper {
+        margin: 0 $sidebarActivityFeedSelectedItemSpacingHorizontal;
+        padding-bottom: $sidebarActivityFeedSpacingVertical - $sidebarActivityFeedSelectedItemBorderWidth;
+        background-color: $bdl-light-blue-05;
+        border: $sidebarActivityFeedSelectedItemBorderWidth $bdl-box-blue solid;
+        border-radius: $bdl-border-radius-size-med;
+    }
+
+    .bcs-ActivityThread-content > .bcs-ActivityCard {
+        padding-top: 10px - $sidebarActivityFeedSelectedItemBorderWidth;
```

Comment:
Is this the same 10 pixels as the `top` style in `.bcs-AnnotationActivity-menu`? If so, should it be a variable?